examples/api-samples/src/browser/view/sample-unclosable-view-contribution.ts

```diff
@@ -33,10 +47,46 @@ export class SampleUnclosableViewContribution extends AbstractViewContribution<S
             }
         });
     }
+
+    protected withWidget<T>(widget: Widget | undefined = this.tryGetWidget(), cb: (sampleView: SampleViewUnclosableView) => T): T | false {
+        if (widget instanceof SampleViewUnclosableView && widget.id === SampleViewUnclosableView.ID) {
+            return cb(widget);
+        }
+        return false;
+    }
```

Comment:
minor: I'd move the `protected` methods after the public ones (at the end of the class).